src/BoundaryConditions/field_boundary_conditions.jl

```diff
@@ -73,7 +73,7 @@ Returns boundary conditions for auxiliary fields (fields
 whose values are derived from a model's prognostic fields) on `grid`
 and at `location`.
 
-Keyword arguments specify boundary conditions on the 7 possible boundaries:
+Keyword arguments specify boundary conditions on the 6 possible boundaries:
```

Comment:
So is this 6 or 7? I know it's the immersed also, but the list here lists only 6...

Reply:
I guess there is a field `immersed`, but it doesn't do anything yet. I think it's fine to leave this at `6`.